spec/strava/api/client/endpoints/uploads/create_upload_spec.rb

```diff
@@ -17,4 +17,18 @@
     expect(upload.status).to eq 'Your activity is still being processed.'
     expect(upload.activity_id).to eq nil
   end
+  it 'returns a Strava::Errors::UploadFailed on upload error', vcr: :all do
+    expect do
+      client.create_upload(
+        file: Faraday::UploadIO.new(file, 'application/tcx+xml'),
+        data_type: 'tcx'
+      )
+    end.to raise_error(Strava::Errors::UploadFailed) do |error| # rubocop:disable Style/MultilineBlockChain
```

Comment:
For rubocop run `rubocop -a ; rubocop --auto-gen-config`, don’t suppress inline.